sdk/src/fee.rs

```diff
@@ -31,6 +31,19 @@ pub struct FeeStructure {
     pub compute_fee_bins: Vec<FeeBin>,
 }
 
+/// Return type of calculate_fee(...)
+#[derive(Debug, Default, Clone, Eq, PartialEq)]
+pub struct FeeDetails {
+    pub transaction_fee: u64,
+    pub prioritization_fee: u64,
+}
+
+impl FeeDetails {
+    pub fn sum(&self) -> u64 {
```

Comment:
```suggestion
    pub fn total_fee(&self) -> u64 {
```

better self-documentation